common/src/main/scala/explore/components/TileController.scala

```diff
@@ -126,20 +127,50 @@ object TileController {
               case l                               => l
             }
 
+        val layouts = updateResizableState(p.layoutMap.get)
+
         ResponsiveReactGridLayout(
           width = p.gridWidth.toDouble,
           margin = (Constants.GridRowPadding, Constants.GridRowPadding),
           containerPadding = (Constants.GridRowPadding, 0),
           rowHeight = Constants.GridRowHeight,
           draggableHandle = s".${ExploreStyles.TileTitleMenu.htmlClass}",
-          onLayoutChange =
-            (_: Layout, b: Layouts) => storeLayouts(p.userId, p.section, b, debouncer)(p.ctx),
-          layouts = updateResizableState(p.layoutMap.get),
+          onBreakpointChange = (bk: BreakpointName, _: Int) => bn.setState(bk.some),
+          onLayoutChange = (_: Layout, b: Layouts) =>
+            bn.value.flatMap { bn =>
+              val le = b.layouts.find(_.name.name === bn.name).map(_.layout)
+
+              // Store the current layout in the state for debugging
+              le.map { l =>
+                currentLayout.modState(breakpointLayout(bn).replace(l))
+              }
+            }.getOrEmpty *>
+              storeLayouts(p.userId, p.section, b, debouncer)(p.ctx),
+          layouts = layouts,
           className = p.clazz.map(_.htmlClass).orUndefined
         )(
           p.tiles.map { t =>
             <.div(
               ^.key := t.id.value,
+              // Show tile proprties on the title if enabled
+              bn.value
+                .flatMap(currentLayout.value.get)
+                .flatMap { case (_, _, l) =>
+                  l.l
+                    .find(_.i === t.id.value)
+                    .map { i =>
+                      TagMod.devOnly(
+                        <.div(
+                          ^.cls := "rgl-tile-overlay",
+                          s"id: ${i.i} x: ${i.x} y: ${i.y} w: ${i.w} h: ${i.h}${i.minH.toOption
+                              .foldMap(m => s" minH: $m")}${i.maxH.toOption
+                              .foldMap(m => s" maxH: $m")}${i.minW.toOption
+                              .foldMap(m => s" minW: $m")}${i.maxW.toOption.foldMap(m => s" maxW: $m")}"
+                        )
+                      )
+                    }
+                }
+                .getOrElse(EmptyVdom),
```

Comment:
This debug information will always show in dev mode?

Reply:
Nope, it is hidden via css too. I don't expect we'd use this very often

Reply:
Ah. OK. It seemed pretty special purpose. :)